jslib/locomote.js

```diff
@@ -244,5 +309,5 @@
     }
   };
 
-  return Locomote;
+  return { Client: Locomote, Monitor: SyncMonitor };
```

Comment:
This would break existing implemententation. Possibly should syncMonitor be a different javascript file